### PR TITLE
Orders table improvement

### PR DESCRIPTION
**Pull request by mhxynh**
**Ticket:** [ATS-44: Improve Orders](https://cs375-pos-group-project.atlassian.net/browse/ATS-44?atlOrigin=eyJpIjoiNDIzMDQ5ZDAyOGFlNGI0MDhkZmE3NWU3MTg3ZGExOTkiLCJwIjoiaiJ9)

- Changed Items from JSON object into JSON array (will revisit to work out possible junction table)
- Added new columns to orders including discount, tips, total, payment status, and date ordered

### CODE REVIEW

**Review by nhokzminh698 — APPROVED** [review]

File: setup.sql [hunk 1]
```diff
@@ -34,8 +34,12 @@ CREATE TABLE cart_item (
 --wip
 CREATE TABLE orders (
 	id SERIAL PRIMARY KEY,
-	items JSON, -- will be changed later to something that can be an array of order_items
-	subtotal DECIMAL(10,2) -- for development use only, will be removed later
+	items JSON[], -- revisit for possible junction table implementation
```
Comment (gitnamehere) [inline setup.sql:37]: I think we should keep it as JSON for now and revisit this so it doesn't break current functionality

File: setup.sql [hunk 2]
```diff
@@ -34,8 +34,12 @@ CREATE TABLE cart_item (
...
+	subtotal DECIMAL(10,2),
+	discount DECIMAL(10,2)
```
Comment (gitnamehere) [inline setup.sql:39]: forgot the comma

File: setup.sql [hunk 3]
```diff
@@ -34,8 +34,12 @@ CREATE TABLE cart_item (
...
+	total DECIMAL(10,2),
+	payment_status BOOLEAN,
+	date_ordered DATE NOT NULL,
```
Comment (gitnamehere) [inline setup.sql:42]: remove the comma in the end 
(also should probably comment this out or remove the NOT NULL part so earlier functionality won't break
though if you are going to comment this out, make sure to remove the comma in payment_status)

**Review by gitnamehere — CHANGES_REQUESTED** [review]

**Review by gitnamehere — APPROVED** [review]
looks good to me!

**Pull request merged**